data/zilliqa/assets/zil1h7ke45su2dem4zrpny7sj9ls80jn38228pdx0r/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Genesis Particle",
+  "description": " Beanterra Genesis Particle - Diverse and lovable creatures that roam the realm filled with lush landscapes, snow-capped mountains, and deep blue oceans.",
+  "symbol": "GP",
+  "web": "https://beanterra.io",
+  "email": "hello@greenbeanz.io",
+  "whitepaper": "https://litepaper.greenbeanz.io/",
+  "product": true,
+  "publicTeam": true,
```

Comment:
remove

Reply:
have removed web,email,whitepaper,product and publicTeam

Reply:
I just said to remove product and public team, not the rest

Reply:
have put back web, email, whitepaper